terraform_route_53/cyber.dhs.gov.tf

```diff
@@ -36,11 +36,24 @@ resource "aws_route53_record" "root_CAA" {
   type    = "CAA"
   ttl     = 300
   records = [ "0 issue \"letsencrypt.org\"",
+              "0 issue \"amazon.com\"",
               "0 issuewild \";\"",
               "0 iodef \"mailto:NCATSSecurity@hq.dhs.gov\""
             ]
 }
 
+# This DNS record gives Amazon Certificate Manager permission to
```

Comment:
@felddy, I wasn't sure if you'd prefer this CNAME record here or [here](https://github.com/cisagov/cyhy_amis/blob/develop/terraform_egress_pub/cloudfront.tf).

Reply:
Either place makes sense.  I'd leave it where it is for now.  Thanks for asking.